### PR TITLE
Added optional user id to ping when using discord notifications.

### PR DESCRIPTION
**Pull request by JadeFoXx**
Users can insert a discord user id into the discord_config..json to be pinged by the notification message.
Implemented in a way so it does not break if an old version of the discord_config.json is being used.
Updated Readme to reflect these changes.

### CODE REVIEW

File: notifications/providers/discord.py [hunk 1]
```diff
@@ -20,13 +20,15 @@ def __init__(self):
                 self.config = json.load(json_file)
                 if self.config["webhook_url"]:
                     self.webhook_url = self.config["webhook_url"]
+                    self.user_id = self.config.get("user_id", "N/A")
```
Comment (Hari-Nagarajan) [inline notifications/providers/discord.py:23]: can this just be 
```
self.config.get("user_id")
```
then self.user_id would be None if its left empty.

Comment (JadeFoXx) [reply notifications/providers/discord.py:23]: I use .isdigit() to check if the user has entered a user id.
Leaving self.user_id as None would throw an error.

Comment (Hari-Nagarajan) [reply notifications/providers/discord.py:23]: Ah ok.

**Review by Hari-Nagarajan — DISMISSED** [review]
LGTM

**Review by WillsB3 — APPROVED** [review]

**Review by Hari-Nagarajan — APPROVED** [review]
LGTM

**Pull request merged**